pkg/router/template/configmanager/haproxy/manager.go

```diff
@@ -658,11 +658,12 @@ func (cm *haproxyConfigManager) commitRouterConfig() {
 	cm.commitTimer = nil
 	cm.lock.Unlock()
 
-	// [Re]Adding a blueprint pool route triggers a router state change.
-	// And calling Commit ensures that the config gets written out.
+	// Adding (+removing) a new blueprint pool route triggers a router state
+	// change. And calling Commit ensures that the config gets written out.
 	route := createBlueprintRoute(routeapi.TLSTerminationEdge)
-	route.Name = fmt.Sprintf("%v-1", route.Name)
+	route.Name = fmt.Sprintf("%v-temp-%d", route.Name, time.Now().Unix())
```

Comment:
s/%v/%s/ ?

Reply:
can't we use the route UUID and resource version instead of timestamp?

Reply:
Sounds nice to me, what do you think @ramr?

Reply:
Sure that will work too - will do.